pyensembl/normalization.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2016. Mount Sinai School of Medicine
```

Comment:
https://github.com/tavinathanson/banana saves the day by informing me that this code is not new!